### DISCUSSION TITLE
SAM2 support ?

### DISCUSSION THREAD

**Discussion by web2299** [light-and-ray/sd-webui-replacer#93] [General]
Hi , are you going to support the new segment anything 2 models https://github.com/facebookresearch/segment-anything-2

thank you in advance !

**Comment by negative1ne** [comment]
Yes, I was wondering the same thing also.

**Comment by light-and-ray** [comment]
As I understand, it's specific for videos. Grounding sam2 too https://github.com/IDEA-Research/Grounded-SAM-2

Maybe I will add it for video tab, but I'm not sure does it worth and how much vram it requires; doesn't it work okay with scene changes. Btw I will add a complicated convenient ui especially for video for replacing in semi-automatic mode step by step, with manual mask and the first frame correction soon
